test-files/cases/attribute/unifying/highway-2888/Expected.osm

```diff
@@ -4709,6 +4709,55 @@
         <tag k="hoot:status" v="1"/>
         <tag k="hoot:id" v="-1"/>
     </node>
+    <way visible="true" id="-1000" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-111"/>
+        <nd ref="-109"/>
+        <nd ref="-108"/>
+        <nd ref="-107"/>
+        <nd ref="-135"/>
+        <nd ref="-1035"/>
+        <tag k="length" v="141.8"/>
+        <tag k="source:datetime" v="2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:id" v="-1000"/>
+    </way>
```

Comment:
Way `-1000` is an issue that I'm not sure why is happening.  The way joiner should joint these two.  I'll look at it further.
![2888-way-1000](https://user-images.githubusercontent.com/13385275/53366818-1cf84680-390a-11e9-9e2b-94093577224b.png)

Reply:
Agreed.  The way joiner being used for Attribute Conflation has a lot of special cases added to it, so its possible one of those is preventing it from happening.